Since musl 1.2.5, basename(3) is strict-POSIX compliant, and
needs #include <libgen.h>, instead of the GNU version included
with <string.h>

--- a/audisp/plugins/zos-remote/zos-remote-config.c
+++ b/audisp/plugins/zos-remote/zos-remote-config.c
@@ -32,6 +32,7 @@
 #include <ctype.h>
 #include <unistd.h>
 #include <stdlib.h>
+#include <libgen.h>
 #include "zos-remote-log.h"
 
 /* Local prototypes */
